webpack.config.client.js

```diff
@@ -2,11 +2,17 @@
 const AssetsPlugin = require('assets-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const webpack = require('webpack');
+const dotenv = require('dotenv');
 const { getClientEnvVars } = require('./src/clientEnvVars');
-const getEnv = require('./src/server/env');
+
+const DOT_ENV_CONFIG = dotenv.config();
+
+if (DOT_ENV_CONFIG.error) {
+  throw DOT_ENV_CONFIG.error;
+}
 
 module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
-  const APP_ENV = process.env.APP_ENV || 'live';
+  const APP_ENV = DOT_ENV_CONFIG.parsed.APP_ENV || 'live';
```

Comment:
Is there a reason we're not just doing `process.env.APP_ENV` now?

Reply:
Errrrrr the reasons it changed are no more ... Great spot, I'll flip it back